frontend/pages/MyProfile/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import Portal from '@shopgate/pwa-common/components/Portal';
+import * as portals from '@shopgate/user/constants/Portals';
+import UserForm from '@shopgate/user/components/UserForm';
+import styles from './style';
+
+/**
+ * The User Profile component.
+ */
+class MyProfile extends Component {
+  static propTypes = {
+    View: PropTypes.func.isRequired,
+  }
+
+  static contextTypes = {
+    i18n: PropTypes.func,
+  };
+
+  /**
+   * @return {string}
+   */
+  get title() {
+    const { __ } = this.context.i18n();
+    return __('profile.title');
+  }
+
+  /**
+   * @return {*}
+   */
+  render() {
+    const { View } = this.props;
+    return (
+      <View title={this.title}>
```

Comment:
In the design the title is actually empty in iOS and only the headline is shown underneath.

Reply:
You removed it pi with addresses